src/Operators/finitedifference.jl

```diff
@@ -2148,6 +2148,12 @@ function apply_stencil!(field_out, bc)
     lw = left_interor_window_idx(bc, space, lbw)
     ri = right_idx(space)
     rw = right_interor_window_idx(bc, space, rbw)
+    if !(typeof(ri) == typeof(rw))
+        error("oops")
```

Comment:
I am always in favor of more informative error messages :)

Reply:
Haha, yeah, this is just to test if this condition is generally true or not. I'll be removing it :)

Reply:
this will always be true because the space is checked before the broadcasted expression is materialized